src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationService.kt

```diff
@@ -84,7 +85,8 @@ class AdjudicationService(
     pageRequest: Pageable,
     adjudicationFilter: AdjudicationFilter,
   ): Page<AdjudicationIdResponse> {
-    return Page.empty()
+    return adjudicationIncidentPartyRepository.findAll(AdjudicationSpecification(adjudicationFilter), pageRequest)
+      .map { AdjudicationIdResponse(adjudicationNumber = it.adjudicationNumber!!, offenderNo = it.offenderBooking!!.offender.nomsId) }
   }
```

Comment:
we can use !! since we only find parties that have adjudication numbers and they must be prisoners (rather than staff)